docs/_layouts/default.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="chrome=1">
+  <title>Docs: {{ site.title | default: site.github.repository_name }} by {{ site.github.owner_name }}</title>
+
+  <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+  <meta name="viewport" content="width=device-width">
+  <!--[if lt IE 9]>
+  <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+</head>
+<body>
+<header>
+  <h1>
+    <a href="http://uw-madison-doit.github.io/angularjs-portal/" aria-label="go to docs home page">{{ site.title | default: site.github.repository_name }}</a>
+  </h1>
+  <p>{{ site.description | default: site.github.project_tagline }}</p>
+
+  {% if site.github.is_project_page %}
+  <p class="view">
+    <a class="button" href="{{ site.github.repository_url }}">
+      <img src="../img/octocat-small.png">
+      View on GitHub
+    </a>
+  </p>
+  {% endif %}
+
+</header>
+<section>
+
+  {{ content }}
+
+</section>
+<footer>
+  {% include sidebar.html %}
```

Comment:
In retrospect, I could just rename the file "footer-aside.html" or something along those lines, for the sake of clarity.